.github/workflows/build-test-measure.yml

```
@@ -657,8 +663,10 @@ jobs:
 
   build-zip:
     name: 'Build: ${{ matrix.build }} build ZIP'
-    # Only run if it is not a draft PR and the PR is not from a forked repository.
-    if: ( github.event.pull_request.draft || github.event.pull_request.head.repo.fork ) == false
+    # Only run if the PR was not authored by Dependabot and it is not a draft or not from a fork.
+    if: >
+      ( github.event.pull_request.draft || github.event.pull_request.head.repo.fork ) == false &&
```

This is kinda hard to read. Maybe this would be simpler:

```suggestion
      github.event.pull_request.draft == false && github.event.pull_request.head.repo.fork == false &&
```

Same for other instance below.

---

Done in https://github.com/ampproject/amp-wp/pull/6253/commits/9549afcea0fb376890eb4932b3e2e8ace75c0e3a.